portfolio/src/main/webapp/template.html

```diff
@@ -65,56 +82,88 @@ <h2 class="heading">About Me</h2>
-                                    <p><a class="more-link" href="https://collegespreadsheets.herokuapp.com" target="_blank"><i class="fas fa-external-link-alt"></i>Find out more</a></p>
+                                    <h3 class="title">
+                                        <a href="https://collegespreadsheets.herokuapp.com" target="_blank">College Spreadsheets- A data driven approach to finding the right college</a>
+                                    </h3>
+                                    <p class="mb-2">React.js web app for high school students that generates personalized recommendations 
+                                        based on inputted preferences. Pulled data from U.S. Dept of Education's API into a Flask-RESTful 
+                                        backend that uses python and pandas to filter, clean and email spreadsheets to students.
+                                    </p>
+                                    <p>
+                                        <a class="more-link" href="https://collegespreadsheets.herokuapp.com" target="_blank">
+                                            <i class="fas fa-external-link-alt"></i>
+                                            Find out more
+                                        </a>
+                                    </p>
                                 </div><!--//desc-->                          
                             </div><!--//item-->
                             
                             <div class="item row">
                                 <a class="col-md-4 col-12" href="#" target="_blank">
-                                <img class="img-fluid project-image rounded shadow-sm" src="assets/images/projects/microcontroller.jpg" alt="Microcontroller" />
+                                    <img 
+                                        class="img-fluid project-image rounded shadow-sm" 
+                                        src="assets/images/projects/microcontroller.jpg" 
+                                        alt="Microcontroller" />
                                 </a>
                                 <div class="desc col-md-8 col-12">
-                                    <h3 class="title"><a href="#" target="_blank">Multicycle Processor</a></h3>
-                                    <p class="mb-2">For the final project of Digital Electronics and Computer Engineering at Harvey Mudd, I built a multicycle RISC-V processor on an FPGA. Involved multiple finite state machines and understanding of microcontroller design from logic gates to the ARM instruction set.  </p>
+                                    <h3 class="title">
+                                        <a href="#" target="_blank">Multicycle Processor</a>
```

Comment:
Are these `href="#"` links part of the template?

Reply:
Yes- I am still considering whether I want to remove or link to something